### PR TITLE
Initial dev mode param implementation

### LINKED ISSUE #88: Add a `dev` param
This would initially control the default view (to go straight to details), and could also be used to add in a test selection list (and thus allow us to remove InteractiveRunner)

### PR DESCRIPTION
**Pull request by bgrins**
Fixes #88. For now this just switches to the details view. I'd like to expand it over time so it has enough features to remove InteractiveRunner.html.

### CODE REVIEW

**Review by flashdesignory — APPROVED** [review]

File: resources/main.mjs [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { params } from "./params.mjs";
 
 // FIXME(camillobruni): Add base class
 class MainBenchmarkClient {
+    dev = false;
```
Comment (rniwa) [inline resources/main.mjs:9]: "dev" is a really generic name. Can we instead call it `isInDeveloperMode`?

Comment (bgrins) [reply resources/main.mjs:9]: Maybe we meet in the middle - `devMode` or `developerMode`? More descriptive than `dev` but also still <= 2 words like other params.

Comment (bgrins) [reply resources/main.mjs:9]: Renamed to developerMode in the latest push - lmk if that works for you

File: resources/main.mjs [hunk 2]
```diff
@@ -85,7 +87,11 @@ class MainBenchmarkClient {
         this._populateDetailedResults(results.formattedValues);
         document.getElementById("results-with-statistics").textContent = results.formattedMeanAndDelta;
 
-        this.showResultsSummary();
+        if (this.dev) {
```
[the remark this answers is not in the document]
Comment (bgrins) [reply resources/main.mjs]: Ah I had been running `npm run format` and it didn't make this change. If I run `npm run pretty:fix` it does but also a bunch of other changes which makes me think maybe the format command needs to be running lint first and then prettier or something? cc @flashdesignory

[the remark this answers is not in the document]
Comment (bgrins) [reply resources/main.mjs]: I'll fix this manually in the commit and we can figure out the tooling separately

**Review by rniwa — APPROVED** [review]

**Pull request merged**